Automattic/jetpack
# VideoPress: Add poster control

#### Changes proposed in this Pull Request:

* Adds a control to manage a custom Poster image for the video

#### Other information:


#### Jetpack product discussion


NA

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Add a new block and add a video (either by upload or URL)
* Check the new Poster image control in the Toolbar

![Captura de tela de 2022-07-01 17-47-48](https://user-images.githubusercontent.com/971483/176967705-026972f3-f6a0-44b1-a496-9a1bf7acef88.png)

* Test defining a custom poster image and confirm it works as expected. The preview image of the video should change
* Test opening the dialog again and removing the image

![Captura de tela de 2022-07-01 17-45-17](https://user-images.githubusercontent.com/971483/176967778-d40c7607-0939-4183-b65c-232f30380ff8.png)

EDIT - I wasn't happy with the result above, because
* We were using the full sized image to show the thumbnail, making it sometimes slow and ugly (browser force resizing)
* We can actually see the image just beneath the menu, in the actual video, so it was redundant.

Here is how it looks now:
![Captura de tela de 2022-07-06 17-24-40](https://user-images.githubusercontent.com/971483/177636706-56f31856-b982-4f6f-afd1-1c53c6b10705.png)




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202532622469874
